spring-hibernate5/src/main/resources/persistence-mysql.properties

```diff
@@ -0,0 +1,13 @@
+# jdbc.X
+jdbc.driverClassName=com.mysql.jdbc.Driver
```

Comment:
When running the FooServiceBasicIntegrationTest, I get the following message in the console:
Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'